### ISSUE TITLE
Undo Sync

### ISSUE THREAD

**Issue by pflajszer** [thesamim/TickTickSync#169]
Could we add instructions on how to undo the sync if possible? I feel like I need some restructuring done but it looks like all my ticktick tasks have been updated and I'd like to revert all that if poissible to sync later.

**Comment by thesamim** [comment]
@pflajszer : Short answer: No. Sorry.

But if you just need to move tasks around, the plugin is designed to handle those moves. If you move from a file that is associated with a List to another file associated with another List, the plugin will also move it in TickTick to the new list. If the new file is not associated with a List, then the task will end up in the TickTick Inbox. If you move a task in TickTick from one List to another, the plugin will automatically move that task to an associated File in the Vault.

If I'm misunderstanding the question, please provide more details.

**Comment by pflajszer** [comment]
This can be closed now - I think I achieved what I needed.

I just needed to sync into a particular folder rather than the root of my vault. Something must've been afoot as it was not actually working as expected. I've refreshed the entire thing (rm the plugin, rm the synced files and rm the cached plugin folders, then reinstalled fresh) and works like a charm - touch wood!

The tool is a bit more forgiving than I expected! thanks for all your hard work!
